src/controllers/passwordModule.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const crypto = require('crypto');
+const env2 = require('./config.env');
```

Comment:
I think this line should be `require('env2')('config.env')`

Reply:
👍 done